packages/editor/src/client/atoms/dateTimePicker.tsx

```diff
@@ -88,11 +88,11 @@ export function DateTimePicker({
                 value: new Date()
               },
               {
-                label: t('dateTimePicker.hour', {hour: '6'}),
+                label: t('dateTimePicker.hour', {hour: '5'}),
                 value: () => {
-                  const six = new Date()
-                  six.setHours(6, 0, 0)
-                  return six
+                  const five = new Date()
```

Comment:
Can you for example name it "date" instead?

Reply:
@bishoymelek it's a date object but it's the time, so probably better call it time or timePreset